src/content/contributing/designers/designers.mdx

```diff
@@ -1,30 +1,58 @@
 ---
-title: Designers
+title: Contribution model
 ---
 
-## Contribution models
+### Carbon is an open source design system built and improved through community contribution. If you like Carbon, help us build it.
 
-Welcome and thank you for contributing! The Carbon team is committed to maintaining consistent, quality guidelines. We welcome all feedback, designs, or ideas in order to produce the best possible experience for our users. Below are our contribution models and best practices.
+<AnchorLinks>
 
-Carbon has three different levels of contribution. Click on the contribution level name to read more details about the workflow process for that particular level of contribution.
+- [How to help](#how-to-help)
+- [Standards](#standards)
+- [Repo-specific guidelines](#repo-specific-guidelines)
 
-- [Light](https://github.com/carbon-design-system/carbon-contribution/wiki/1.-Light-Contribution-Model): A small design tweak (i.e. changing a border color)
-- [Medium](https://github.com/carbon-design-system/carbon-contribution/wiki/2.-Medium-Contribution-Model): Adding to/or changing guidelines, or contributing an icon
-- [Heavy](https://github.com/carbon-design-system/carbon-contribution/wiki/3.-Heavy-Contribution-Model): Contributing a whole new component to the system
+</AnchorLinks>
 
-### Submitting an issue
+## How to help
 
-If you are not sure which model your contribution fits into, open an issue in one our repos and we will figure out which model it falls into.
+### Leave feedback
 
-#### Not sure where to open an issue?
+The best place to leave feedback is [GitHub](https://github.com/carbon-design-system/carbon/issues/new/choose). If you're not already a member, it takes just a minute to [join](https://github.com/join). Carbon offers a few feedback templates, which make sure an issue includes all the information necessary for the team to take a look.
 
-All contribution proposals should be opened in the [carbon-contribution repo](https://github.com/carbon-design-system/carbon-contribution). This includes light, medium, and heavy contributions (all components, icons, guideline updates, etc.).
+### Fix a bug
 
-#### Not sure how to submit an issue?
+If you find something that's broken, you can let us know with [an issue](https://github.com/carbon-design-system/carbon/issues/new/choose). If you know how to fix it, and want to make Carbon even better, you can fix the bug and submit a pull request!
```

Comment:
should prob point to the bug template rather than the choose template link., but might be ok since they might have to chose between bug report and accessibility issue.